mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -159,14 +157,21 @@ class GBTClassifier @Since("1.4.0") (
     val numFeatures = oldDataset.first().features.size
     val boostingStrategy = super.getOldBoostingStrategy(categoricalFeatures, OldAlgo.Classification)
 
-    val instr = Instrumentation.create(this, oldDataset)
+    val numClasses: Int = getNumClasses(dataset)
+    if (isDefined(thresholds)) {
```

Comment:
There is no `setThresholds` method, and there are no unit tests off the top of my head.